### PR TITLE
Vc: Update to 1.3.3 and migrate to GitHub.

### PR DESCRIPTION
**Pull request by cjones051073**
#### Description

Vc : Update to 1.3.3 and migrate to GitHub.

FYI @RJVB

- [x] enhancement

macOS 10.13.5 17F77
Xcode 9.4.1 9F2000 

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] checked your Portfile with `port lint`?
- [x] tried a full install with `sudo port -vst install`?

### CODE REVIEW

File: math/Vc/Portfile [hunk 1]
```diff
@@ -3,9 +3,11 @@
 PortSystem          1.0
 PortGroup           cmake 1.0
```
[the remark this answers is not in the document]
Comment (cjones051073) [reply math/Vc/Portfile]: Will do

File: math/Vc/Portfile [hunk 2]
```diff
@@ -3,9 +3,11 @@
...
 PortGroup           muniversal 1.0
+PortGroup           github 1.0
+PortGroup           cxx11 1.1
 
-name                Vc
-version             0.7.4
+github.setup        VcDevel Vc 1.3.3
+github.tarball_from releases
 categories          math devel
 license             LGPL-3
```
[the remark this answers is not in the document]
Comment (cjones051073) [reply math/Vc/Portfile]: Yeah, I also saw that. Should we just change it to 'BSD' ?

File: math/Vc/Portfile [hunk 3]
```diff
@@ -22,20 +24,12 @@ long_description \
...
-                    # Attention: the download directory changes for each release ...
-master_sites        http://code.compeng.uni-frankfurt.de/attachments/download/183/
-
-checksums           rmd160  0878034cd1b78ad0429974c81502d2912c6f3e1b \
-                    sha256  a4e5c5d7b51b4b04e6ef87e28a5b8b0108c9788739b044f96fb64f274d249063
+checksums           rmd160  abb533dc9dfdcb0285e0f5438fc67f5724d3fb1f \
+                    sha256  08c629d2e14bfb8e4f1a10f09535e4a3c755292503c971ab46637d2986bdb4fe \
+                    size    781361
 
 post-extract        { file mkdir ${workpath}/build }
```
[the remark this answers is not in the document]
Comment (cjones051073) [reply math/Vc/Portfile]: will remove.

**Comment by cjones051073** [comment]
@RJVB I would suggest any changes to the build system or adding variants should be the focus of a separate PR. This one is (just) about updating the version (which I need to then use this port with root6).

**Comment by RJVB** [comment]
> @RJVB I would suggest any changes to the build system or adding variants should be the focus of a separate PR.

Maybe for a variant, but disabling the building of the tests doesn't change anything in which files are installed. Idem for the examples, and worse, disabling those shouldn't even affect the possibility to run `port test`.

Re: license: yeah, I think it should be changed to `BSD` as there appears to be no better option. Or maybe the "3 clause" BSD license is the standard BSD license anyway.

**Comment by cjones051073** [comment]
Ok, will update the license to BSD. 

I will still though, if you don't mind, prefer to leave any other changes to the build, disabling tests etc. To be implemented by yourself as the subject of a different PR.

**Comment by RJVB** [comment]
OK then. LGTM otherwise.

**Comment by cjones051073** [comment]
Thanks. I won’t merge myself, as I think its better for a second MP member to first take a look and then do it.

**Comment by pmetzger** [comment]
LGTM. I'll merge.

**Comment by RJVB** [comment]
That's up to you of course, but this is just a basic upgrade and it looks like root6 may be the only dependent...

**Comment by cjones051073** [comment]
@pmetzger Thanks !

@RJVB I guess its just a point of principle. As a committer if i go to the effort of preparing a PR it is because I want review. I think in these cases merging it myself just seems wrong, better to leave it to another MP member.

**Pull request merged**